content/docs/command-reference/params/index.md

```diff
@@ -204,6 +208,9 @@ SET = {4, 5, 6}
 TUPLE = (10, 100)
 NONE = None
 
+# Complex expressions will be ignored.
+DICT_EXP = dict(a=1, b=2)
```

Comment:
Is that correct? Will they be ignored or will there be some sort of error @daavoo ? Thanks

Reply:
It is currently ignored. I'm thinking about sending a P.R. to at least log a warning